### DISCUSSION TITLE
<Tree issue>ref-key of parent node in Tree is always stay String_undefined even after values given to data-key

### DISCUSSION THREAD

**Discussion by akomal759** [rsuite/rsuite#3093] [Q&A]
I have explicitly assigned values to data-key but still i get the parent node's ref-Key is "String_undefined" but got the right values for their children. Can anyone help me in this matter?

**Comment by SevenOutman** [comment]
Hi @akomal759 , I just saw you created an issue #3094 for the very same topic, so I'm going to lock this discussion. Let's continue this  topic in that issue.
